pkgs/applications/science/math/sage/threejs-sage.nix

```
@@ -12,7 +12,8 @@ stdenv.mkDerivation rec {
   };
 
   installPhase = ''
-    mkdir -p $out/lib/node_modules/three
-    cp -r build version $out/lib/node_modules/three
+    mkdir -p "$out/lib/node_modules/three/$(cat version)"
+    cp version "$out/lib/node_modules/three"
+    cp -r build/* "$out/lib/node_modules/three/$(cat version)"
```

You could also do

```suggestion
    mkdir -p "$out/lib/node_modules/three/"
    cp version "$out/lib/node_modules/three"
    cp -r build "$out/lib/node_modules/three/$(cat version)"
```

saving a glob and a `cat`. The main downside of the other version is that a glob misses files that start with a dot. It likely doesn't make a difference in practice here, so if you prefer the glob version that's also fine by me.

Sorry for not noticing this during the first round of review.

---

Ooh, that's a good point, thanks!